Tella/Scenes/Settings/Views/Servers/GDrive/CreateDriveFolder.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  CreateDriveFolder.swift
+//  Tella
+//
+//  Created by gus valbuena on 5/21/24.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import SwiftUI
+
+struct CreateDriveFolder: View {
+    @State var fieldContent : String = ""
+    @State var isValid : Bool = false
+    var body: some View {
+        ContainerView {
+            VStack(spacing: 20) {
+                Spacer()
+                ServerConnectionHeaderView(
+                    title: "Create new folder",
+                    subtitle: "Your reports will be uploaded to a new folder on your Google Drive. Choose a name for this folder here."
+                )
+                TextfieldView(fieldContent: $fieldContent,
+                              isValid: .constant(true),
```

Comment:
Could you try `$isValid` instead of adding `.onChange`

                TextfieldView(fieldContent: $fieldContent,
                              isValid: $isValid,
                              shouldShowError: .constant(false),
                              fieldType: .text,
                              placeholder: "Folder name")
                    .padding(.vertical, 12)